packages/react-components/src/components/utils/longPress.ts

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Did you look at https://github.com/minwork/use-long-press ?

The benefit is that it seems to have enough usage that it'll have better testing across platforms / browsers etc.

Reply:
They had an open issue where their peer dependency was tied to react < 17 so I didn't want to bind us further to that issue - but they just fixed it as of yesterday!
![image](https://user-images.githubusercontent.com/2684369/140101519-4ea5bfcb-1657-4064-957d-33b07304b595.png)

Reply:
Updated to use-long-press, much cleaner, thanks!